go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/xeipuuv/gojsonschema v1.2.0
-	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83
+	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
```

Comment:
Is there a way to prevent this update in this PR? I'm trying to figure out what's necessary for the name change vs. what should be pulled out into a separate PR. Changes to /x/crypto might be best as a separate review item if at all possible.

Reply:
I can't tell, this has been introduced by `go mod tidy`, so I assume required by transitive dependencies...

Reply:
@ndeloof I agree with @bacongobbler. Lets not change modules that are not part of the PR in question. Do you mind rolling this back?

Reply:
As I said, this update is forced by `go mod`, if I revert this it will be added again any time you run `go mod xx` commands.

Reply:
@ndeloof Looking further https://github.com/oras-project/oras-go/blob/main/go.mod is added in https://github.com/oras-project/oras-go/blob/main/go.mod, so probably picking it up from there.